src/vm/ClrEtwAll.man

```diff
@@ -1624,6 +1631,48 @@
                         </UserData>
                     </template>
 
+                    <template tid="AssemblyLoadStart">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="AssemblyName" inType="win:UnicodeString" />
+                        <data name="AssemblyPath" inType="win:UnicodeString" />
+                        <data name="ParentAssembly" inType="win:UnicodeString" />
```

Comment:
I would not call this "parent" - I think a better name is "requesting assembly". There's a precedence for it here: https://docs.microsoft.com/en-us/dotnet/api/system.resolveeventargs.requestingassembly?view=netcore-3.0#System_ResolveEventArgs_RequestingAssembly